src/fides/api/db/system.py

```diff
@@ -48,27 +52,47 @@ def get_system(db: Session, fides_key: str) -> System:
     return system
 
 
+async def validate_data_labels(
+    db: AsyncSession, sql_model: Base, labels: List[FidesKey]
```

Comment:
technically I'd want to type `sql_model` with something more specific than `Base` since I'm assuming that resource has an `active` property but wasn't sure if this was something I could do 😅 I'm not too familiar with Python type annotations so don't know how complex I can get with the types

Reply:
most things inherit from `Base` so it's technically correct, but I think it might be too broad here, especially since Base isn't guaranteed to have the `active` property.  For now, since you're calling this in one place, I might annotate with the known resources you're calling with, a data category, a data subject, and a use, like:
```
sql_model: Union[Type[DataUse], Type[DataSubject], Type[DataCategory]]
```

The `Type` is there because it's not an instance of a DataUse you're passing in, rather it's just a class definition

Reply:
I have updated this in [37036ac](https://github.com/ethyca/fides/pull/4982/commits/37036ac66b5830b60df303be7bade4136d10cc7b) :D